app/views/animals/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>This will be replaced by React</h1>
```

Comment:
You have both an index ERB and an index React component - I would choose one or the other